docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc

```diff
@@ -7,38 +7,70 @@
+
+[discrete]
+[[elastic-agent-input-configuration-sample-metrics]]
+== Sample metrics input configuration
+
+By default {agent} collects system metrics, such as CPU, memory, network, and file system metrics, and sends them to the default output. For example, to define datastreams for `cpu`, `memory`, `network` and `filesystem` metrics, this is the configuration:
 
 ["source","yaml"]
 -----------------------------------------------------------------------
-- id: unique-system-metrics-id <1>
-  type: system/metrics <2>
-  use_output: default <3>
-  meta:
-    package: <4>
-      name: system
-      version: 0.10.9
-  data_stream:
-    namespace: default <5>
+- type: system/metrics <1>
+  id: unique-system-metrics-id <2>
+  data_stream.namespace: default <3>
+  use_output: default <4>
   streams:
-    - data_stream:
-        dataset: system.cpu <6>
-        type: metrics <7>
-      metricsets: <8>
-        - cpu
-      period: 10s
-      cpu.metrics:
-        - percentages
-        - normalized_percentages
+    - metricsets: <5>
+      - cpu
+      data_stream.dataset: system.cpu <6>
+    - metricsets:
+      - memory
+      data_stream.dataset: system.memory
+    - metricsets:
+      - network
+      data_stream.dataset: system.network
+    - metricsets:
+      - filesystem
+      data_stream.dataset: system.filesystem
 -----------------------------------------------------------------------
 
-<1> A unique ID for the input.
-<2> A generic type describing the data.
-<3> The name of the `output` to use. If not specified, `default` will be used.
-<4> Package specification.
-<5> A user-defined namespace.
-<6> A user-defined dataset. It can contain anything that makes sense to signify the source of the data.
-<7> The type of the data stream.
-<8> Enabled module metricsets.
+<1> A generic type describing the data.
+<2> A unique ID for the input.
+<3> A user-defined namespace.
+<4> The name of the `output` to use. If not specified, `default` will be used.
+<5> The set of enabled module metricsets.
 +
-In the {metricbeat-ref}/metricbeat-module-system.html[System module] there are several options. The `cpu` is just one of them. Its fields can be configured.
\ No newline at end of file
+Refer to the {metricbeat} {metricbeat-ref}/metricbeat-module-system.html[System module] for a list of available options. The metricset fields can be configured.
+<6> A user-defined dataset. It can contain anything that makes sense to signify the source of the data.
+
+[discrete]
+[[elastic-agent-input-configuration-sample-logs]]
+== Sample log files input configuration
+
+To enable {agent} to collect log files, you can use a configuration like the following.
+
+["source","yaml"]
+-----------------------------------------------------------------------
+- type: filestream <1>
+  id: your-input-id <2>
+  streams:
+    - id: your-filestream-stream-id <3>
+      data_stream: <4>
+        dataset: generic
+      paths:
+        - /var/log/*.log
```

Comment:
I believe it would be interesting to also mention referring to the filestream documentation for other available options, like in the Metricbeat example.

Reply:
Good idea. Added via [a1fdb82](https://github.com/elastic/ingest-docs/pull/489/commits/a1fdb82dcd133f785eee59444350a206a72afdde)